app/notifiers/course/forum/post_notifier.rb

```diff
@@ -1,15 +1,36 @@
 # frozen_string_literal: true
 class Course::Forum::PostNotifier < Notifier::Base
-  # To be called when user replied a forum post.
-  def post_replied(user, post)
-    course = post.topic.actable.forum.course
+  # Called when a user replies to a forum post.
+  #
+  # @param[User] User who replied to the forum post
+  # @param[CourseUser] course_user The course_user who replied to the forum post.
+  #   This can be +nil+ in exceptional cases where the administrator posts to a forum.
+  # @param[Course::Discussion::Post] post The post that was created.
+  def post_replied(user, course_user, post)
+    course = post.topic.course
     activity = create_activity(actor: user, object: post, event: :replied)
-    activity.notify(course, :feed)
-    if Course::Settings::ForumsComponent.email_enabled?(course, :post_replied)
+    activity.notify(course, :feed) if course_user && !course_user.phantom?
+
+    if email_nofitication_enabled?(course_user)
```

Comment:
Got typo here. Should be `email_notification_enabled`. Same for the function definition on line 25.